src/embind/embind.js

```diff
@@ -1773,7 +1773,7 @@ var LibraryEmbind = {
             "Make sure to invoke .delete() manually once you're done with the instance instead.\n" +
             "Originally allocated"); // `.stack` will add "at ..." after this sentence
             if ('captureStackTrace' in Error) {
-                Error.captureStackTrace(info.leakWarning, cls.constructor);
+                Error.captureStackTrace(info.leakWarning, Object.getPrototypeOf($$.ptrType).fromWireType);
```

Comment:
Wouldn't `cls.prototype.fromWireType` be the same but shorter?

Reply:
Oh right, because `$$.ptrType` is an instance of `RegisteredPointer` not of C++ class.

Reply:
Actually, looks like it's just a freestanding function: https://github.com/emscripten-core/emscripten/blob/c0f1058cc53736a078e1bfaa42f0f2b8beebbccb/src/embind/embind.js#L1644

Can you try adding `'$RegisteredPointer_fromWireType'` to the `__deps` array and then just using `RegisteredPointer_fromWireType` here?

Reply:
Great! That should be a bit better, because minifier can easily compress long variable names to one-letter ones, but can't do much about properties and function calls :)